frontend/appflowy_flutter/lib/mobile/presentation/database/card/card_property_edit/mobile_field_type_option_editor.dart

```diff
@@ -0,0 +1,197 @@
+import 'package:appflowy/generated/flowy_svgs.g.dart';
+import 'package:appflowy/generated/locale_keys.g.dart';
+import 'package:appflowy/mobile/presentation/database/card/card_property_edit/type_option_widget_builder/type_option_widget_builder.dart';
+import 'package:appflowy/mobile/presentation/widgets/show_flowy_mobile_bottom_sheet.dart';
+import 'package:appflowy/plugins/database_view/application/field/field_type_option_edit_bloc.dart';
+import 'package:appflowy/plugins/database_view/application/field/type_option/type_option_data_controller.dart';
+import 'package:appflowy/plugins/database_view/grid/presentation/widgets/header/field_type_extension.dart';
+import 'package:appflowy/plugins/database_view/grid/presentation/widgets/header/type_option/builder.dart';
+import 'package:appflowy_backend/protobuf/flowy-database2/protobuf.dart';
+import 'package:easy_localization/easy_localization.dart';
+import 'package:flowy_infra_ui/flowy_infra_ui.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_bloc/flutter_bloc.dart';
+import 'package:go_router/go_router.dart';
+
+import 'mobile_field_type_list.dart';
+
+class MobileFieldTypeOptionEditor extends StatelessWidget {
+  final TypeOptionController _dataController;
+
+  const MobileFieldTypeOptionEditor({
+    required TypeOptionController dataController,
+    Key? key,
+  })  : _dataController = dataController,
+        super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    return BlocProvider(
+      create: (context) {
+        final bloc = FieldTypeOptionEditBloc(_dataController);
+        bloc.add(const FieldTypeOptionEditEvent.initial());
+        return bloc;
+      },
```

Comment:
Personal preference (formatting is off in my suggestion btw)
```suggestion
      create: (context) => 
          FieldTypeOptionEditBloc(_dataController)
            ..add(const FieldTypeOptionEditEvent.initial()),
```